api/v1/flow/flow.proto

```diff
@@ -375,6 +375,8 @@ enum DropReason {
     NAT46 = 187;
     NAT64 = 188;
     AUTH_REQUIRED = 189;
+    CT_NO_MAP_FOUND = 190;
```

Comment:
@YutaroHayakawa FYI, @ysksuzuki is using the drop number 190 [in his PR](https://github.com/cilium/cilium/pull/23890/files#diff-74cced194f034ecadb39f852432c1822b6ef022065df0cb0d2ff6c718b339d45R378). Whoever is ready to merge first wins. Let me know when you're ready and I'll give you :green_apple: for API.